apps/coscrad-frontend/src/components/dynamic-forms/dynamic-form-elements/multilingual-text-input/actions/multilingual-text-form-reducer.ts

```diff
@@ -0,0 +1,73 @@
+import { IMultiLingualText } from '@coscrad/api-interfaces';
+import { PayloadAction } from '@reduxjs/toolkit';
+import { isUpdateItemRoleAction } from './update-item-role';
+import { isUpdateItemTextAction } from './update-item-text';
+
+export const multiLingualTextReducerWtihNotification =
```

Comment:
This feels wrong, but it's actually conceptually a fairly clean way to deal with the side effect and it works well. I wonder if we could do something similar for the entire form. It might help to simplify a lot of prop drilling of state update callbacks.